README.md

```diff
@@ -23,9 +22,8 @@ else
 Usage
 -----
 ### Initialize
-`
-RazorpayClient client = new RazorpayClient(key, secret);
-`
+`RazorpayClient client = new RazorpayClient(key, secret);`
```

Comment:
Doing this means no syntax highlighting. I suggest we keep it as it is, and throw in `csharp` as the language identifier.